app/models/concerns/module_settings_concern.rb

```diff
@@ -0,0 +1,35 @@
+module ModuleSettingsConcern
```

Comment:
Would it make sense if this file is in the app/models/concerns/course folder instead?